### DISCUSSION TITLE
Cannot find "Depth" tab anywhere... help?

### DISCUSSION THREAD

**Discussion by vdruts** [lllyasviel/stable-diffusion-webui-forge#1981] [Q&A]
Hey guys, I installed Depthmap script plugin, but the Tab is not showing up on the top. The script shows up in the scripts section though. I looked into settings but cannot find that it is hidden anywhere. Any ideas?
![image](https://github.com/user-attachments/assets/4548e8c2-b5b4-4170-a0b3-07573b765a2d)

**Comment by dongxiat** [comment]
https://github.com/thygate/stable-diffusion-webui-depthmap-script << maybe..this extension for A1111 (use gradio version 3) , Forge use gradio version 4 so the tab not show up. U need edit some code to show up in gradio version 4

**Reply by LDSUIQ** [reply]
> https://github.com/thygate/stable-diffusion-webui-depthmap-script << maybe..this extension for A1111 (use gradio version 3) , Forge use gradio version 4 so the tab not show up. U need edit some code to show up in gradio version 4

What code needs to be changed? Please give an example or expand on your answer.

**Comment by vdruts** [comment]
Any idea what I can do?

**Reply by ArnorWing** [reply]
Install A1111 and use that extension there
